src/File/MoveUploadedFileInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Filter\File;
+
+use function move_uploaded_file;
+
+final class MoveUploadedFileInterface implements UploadedFileMoverInterface
```

Comment:
class with `Interface` suffix?

Reply:
Oh automatic renaming without chercking. Will fix it

- [ ] Rename class